src/pages/DiscordPage/DiscordPage.tsx

```diff
@@ -59,6 +59,16 @@ export const DiscordPage = () => {
       const result = await generateCircleApiKey({
         circle_id: Number(circleIdParam),
         name: 'discord-bot',
+        read_circle: true,
+        update_circle: true,
+        read_nominees: true,
+        create_vouches: true,
+        read_pending_token_gifts: true,
+        update_pending_token_gifts: true,
+        read_member_profiles: true,
+        read_epochs: true,
+        read_contributions: true,
+        create_contributions: true,
```

Comment:
Be sure to add the `manage_users` permission here.

Reply:
Oh, totally overlooked that one. Also added `read_discord`, not totally sure it'll be necessary though.

https://github.com/coordinape/coordinape/pull/1955/commits/97d2b018e0c2b864f568539b8f09ea752f0bfc8a